### DISCUSSION TITLE
使用doris官网的SQLServer同步实例执行报错提示 Unknown operation sqlserver-sync-database，少配置了吗

### DISCUSSION THREAD

**Discussion by echobigdata** [apache/doris#24542] [A - General / Q&A] [answered]
flink1.16.2
flink-sql-connector-sqlserver-cdc-2.4.1.jar
mssql-jdbc-9.4.1.jre8.jar
flink-doris-connector-1.16-1.4.0.jar

这是添加的jar包，然后执行官网的案例，提示Unknown operation sqlserver-sync-database
![image](https://github.com/apache/doris/assets/97231854/e16b4d4f-3156-47fd-a7f9-4a578b024f77)
![微信图片_20230918162223](https://github.com/apache/doris/assets/97231854/84d1af05-db6c-4724-b448-cd49597d74d1)

**Comment by ixzc** [comment, marked as answer]
You need to use the latest flink-doris-connector package : https://repository.apache.org/content/repositories/snapshots/org/apache/doris/

**Comment by echobigdata** [comment]
> 您需要使用最新的 flink-doris-connector 包：https://repository.apache.org/content/repositories/snapshots/org/apache/doris/

感谢！
